lib/mix/lib/mix/release.ex

```diff
@@ -425,10 +432,19 @@ defmodule Mix.Release do
   end
 
   defp build_variables(release) do
+    variables =
+      for {_, properties} <- release.applications,
+          not Keyword.fetch!(properties, :otp_app?),
+          uniq: true,
+          do: {'RELEASE_LIB', properties |> Keyword.fetch!(:path) |> :filename.dirname()}
+
+    [{'ERTS_LIB_DIR', erts_lib_dir(release.erts_source)} | variables] |> IO.inspect()
```

Comment:
By using ERTS_LIB_DIR, it means we need to set another variable as --boot-bar when assembling the release, no?